script.js

```diff
@@ -1 +1,152 @@
+//yehey 하이
+const todoForm = document.querySelector(".submitContainer"),
+    todoInput = todoForm.querySelector("input"),
+    todoList = document.querySelector(".todoListContainer"),
+    completeList = document.querySelector(".completeContainer");
 
+const TODOS_LS = "todos";
+let todos =[];    //여러개의 할 일들을 담을 배열
+const COM_LS = 'done';
+let done=[];
+
+function loadToDos(){
```

Comment:
```suggestion
function loadToDosFromLocalStorage(){
```

함수의 이름은 필요한 만큼 구체적으로 적어주시면 좋아요~!~!